docs/Q-and-A.md

```diff
@@ -0,0 +1,277 @@
+# Q&A about KERI
+<img src="../images/Keri_logo_color_on_white.png" alt="KERI logo" border="0" width="300">
+
+- [Q&A about KERI](#qa-about-keri)
+    + [Disclaimer](#disclaimer)
+    + [List of questions and definitions](#list-of-questions-and-definitions)
+  * [Knowledge you should be confidently applying](#knowledge-you-should-be-confidently-applying)
+  * [Actions you should be comfortable with](#actions-you-should-be-comfortable-with)
+- [Jump table to categories](#jump-table-to-categories)
+
+Inspired by presentation given and questions asked on the [SSI webinar May 2020](https://ssimeetup.org/key-event-receipt-infrastructure-keri-secure-identifier-overlay-internet-sam-smith-webinar-58/), but also issues raised and the progress made, here on Github (Turn on 'Watch' if you want to be notified of conversations).
+
+Beware: A Q&A is always *work in progress*. Tips & help welcome.
+
+
+### Disclaimer
+None of the respondents in the **open** repo and presentations have been explicitly named as a source, except for ***Samuel M. Smith Ph.D.*** and ***@henkvancann***. If there is no reference added to the answers, then it's Samuel M. Smith who answered the question. Most of the editing is done by @henkvancann, which might have introduced ommission, errors, language glitches and such. Sorry for that, feel free to correct by submitting a pull request (PR).<br/>
```

Comment:
instead of \<br\/\> simply \ is enough if you want to have new line. See: https://github.github.com/gfm/#hard-line-break

Reply:
ah, didn't know, great because \ teaches you where it wasn't necessary too. Thx! Repaired the Q-and-A.md : no <br/> anymore